tests/base/src/test/java/org/wildfly/security/x500/X509PeerCertificateChainEvidenceDecoderTest.java

```diff
@@ -187,6 +191,24 @@ public void testDefaultEvidenceDecoder() throws Exception {
         assertTrue(sac.authorize()); // "0" does have Admin role
     }
 
+    /**
+     * Verify evidence is being used in realm mapping in ServerAuthenticationContext#assignName.
+     * Test verifies behaviour reported in ELY-2090 is corrected.
+     */
+    @Test
+    public void testEvidenceUsedInServerAuthenticationContextRealmMapping() throws Exception {
```

Comment:
Since this test does not rely on evidence decoders, it would be good to either add it to a different test class or create a new test class (perhaps under `org.wildfly.security.auth.server` or something like that).